Airship documentation update (#507)
rudderlabs/rudderstack-docs: destinations/marketing/airship.mdx

<img src="../../assets/event-stream-destinations/airship-connection-settings.png" alt="Airship connection settings" />

> - **API Key**: Enter your Airship token required by RudderStack to communicate with your Airship project.

Reviewer: App key is only used for track calls maybe we can mention that using callout?
@ujjwal-ab ?

Author: Makes sense. I will add this info.

Reviewer: API key is used for all calls , only track requires app key cc@ujjwal-ab

Reviewer 2: Yes, App key is required in track call only.

Author: I've made the changes. Please review. @utsabc @ujjwal-ab

> - **API Key**: Enter your Airship token which is required by RudderStack to communicate with your Airship project and authenticate all the supported event calls, i.e.[`identify`](#identify), [`track`](#track), and [`group`](#group).

Reviewer 3: ```suggestion
- **API Key**: Enter your Airship token which is required by RudderStack to communicate with your Airship project and authenticate all the supported event calls, that is, [`identify`](#identify), [`track`](#track), and [`group`](#group).
```
